public/aboutus.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Elegant</title>
+  <link rel="stylesheet" href="css/aboutus.css">
+  <link rel="shortcut icon" type="image/x-icon" href="/imgs/icon.ico" />
+  <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+  <link href="https://fonts.googleapis.com/css?family=Playfair+Display" rel="stylesheet">
+</head>
+  <body>
+    <header>
+      <form id="headerForm" class="headerForm" action="" method="post">
+        <input type="submit" name="" value="Login">
+        <input type="submit" name="" value="Sign Up">
+      </form>
+    </header>
+      <div class="burgerAndLogo" id="burgerAndLogo">
+        <div></div>
+    <div><img src="./imgs/logo.png" alt="logo" class="logo"></div>
+    <div class="burgerButton">
+        <div class="slice"></div>
+        <div class="slice"></div>
+        <div class="slice"></div>
+      </div>
+    </div>
+
+    <div class="container" id="container">
+      <h1>About Us</h1>
+      <center><div class="parag"><p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem.</p>
```

Comment:
we shouldn't use center tag anymore!
and in the small screen the text should be wrapped